### PR TITLE
refactor(app): special case 96-channel pipette mount text in drop tip wizard

### PR DESCRIPTION
**Pull request by mjhuff**
Closes RQA-2020

# Overview

Per QA/Design feedback. 

### Current Behavior

Screenshot 2023-12-12 at 11 29 12 AM

### With Fix

Screenshot 2023-12-12 at 11 19 21 AM

# Test Plan
- Launch the drop tip flows on the 96 channel via ODD. Easiest way to currently do this is to throw an estop error after picking up a tip. 

# Changelog
- 96-channel now has special drop tip wizard launch text.

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/DropTipWizard/TipsAttachedModal.tsx [hunk 1]
```diff
@@ -51,6 +51,9 @@ const TipsAttachedModal = NiceModal.create(
       iconColor: COLORS.yellow2,
     }
 
+    const is96Channel = instrumentModelSpecs.displayName.includes('96-Channel')
```
Comment (jerader) [inline app/src/organisms/DropTipWizard/TipsAttachedModal.tsx]: maybe a better way to filter this is if the channels === 96

**Review by jerader — APPROVED** [review]
lgtm! I left a suggestion for filtering for 96-channel

File: app/src/organisms/DropTipWizard/TipsAttachedModal.tsx (hunk 1 above)
Comment (mjhuff) [reply app/src/organisms/DropTipWizard/TipsAttachedModal.tsx]: Brilliant, thanks!

Comment (koji) [reply app/src/organisms/DropTipWizard/TipsAttachedModal.tsx]: A few existing `is96Channel` using `instrumentModelSpecs.channel === 96` so maybe good.
But' I haven't worked on 96 related components, so to me this change makes sense.

**Review by koji — APPROVED** [review]
lgtm

**Pull request merged**